packages/server/src/automations/steps/delay.ts

```diff
@@ -14,6 +14,7 @@ export const definition: AutomationStepSchema = {
   description: "Delay the automation until an amount of time has passed",
   stepId: AutomationActionStepId.DELAY,
   internal: true,
+  canLoop: false,
```

Comment:
Just thinking about this - I would argue this is a bit of a feature flag, it might be useful to do something like we've done with the datasources and actually have a `features` array that we add to automations, e.g.

`features: [AutomationFeature.LOOPING]` - this way if the feature is not present we can assume it doesn't support looping. This is also quite a bit more re-usable if we ever want to be able to turn off features for other blocks as well.